docs/integrations/sources/jira.md

```diff
@@ -126,7 +126,7 @@ The Jira connector should not run into Jira API limitations under normal usage.
 
 | Version | Date       | Pull Request                                                | Subject                                                                                                                 |
 |:--------|:-----------|:------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------------------|
-| 0.3.6   | 2023-04-10 | [\#24636](https://github.com/airbytehq/airbyte/pull/24636)  | Removed Connector Domain Pattern from Spec                                                                              |
+| 0.3.6   | 2023-04-10 | [\#24636](https://github.com/airbytehq/airbyte/pull/24636)  | Remove pattern for domain check in the specification                                                                          |
```

Comment:
Why do we have this filed modified? It's related to the other connector (source-jira).